internal/blueprint/blueprint.go

```diff
@@ -103,6 +121,14 @@ func (b *Blueprint) GetPackages() []string {
 	for _, group := range b.Groups {
 		packages = append(packages, "@"+group.Name)
 	}
+
+	if kc := b.Customizations.GetKernel(); kc != nil && kc.Name != "" {
+		kpkg := Package{Name: b.Customizations.Kernel.Name}
+		packages = append(packages, kpkg.ToNameVersion())
+	} else if !b.hasKernelPackage() { // no Kernel specified; add default (unless already in packages)
+		kpkg := Package{Name: "kernel"}
```

Comment:
Should we perhaps make `DefaultKernel` a property of each image type and/or distro?

Another option would be to move the logic of appending the kernel to the package list into each distro package, we already have logic there to append packages to the default package list based on the blueprint.

Reply:
I started adding `supportedKernels` as a property of the image types but realised that was going down a path to bigger changes.  The `DefaultKernel` is a good middle ground there.  I'll make the change.

Reply:
Feel free to just keep this as is though. Maybe in the future we get image types without kernels, or we get distores with different default kernels, but we can refactor when and if that happens, it won't change the API.